### ISSUE TITLE
camera_calibration/calibrator.py: calibration dump fails because self.size is not set

### ISSUE THREAD

**Issue by SJamieson** [ros-perception/image_pipeline#501]
OS: Ubuntu 18.04
Behaviour: Try calling do_calibration with dump=True, receive the following error:
```
Traceback (most recent call last):
  File "/opt/ros/melodic/lib/python2.7/dist-packages/camera_calibration/camera_calibrator.py", line 269, in on_mouse
    self.c.do_calibration()
  File "/opt/ros/melodic/lib/python2.7/dist-packages/camera_calibration/calibrator.py", line 1226, in do_calibration
    pickle.dump((self.is_mono, self.size, self.good_corners),
AttributeError: 'StereoCalibrator' object has no attribute 'size'
```

Desired behaviour: The size that is set in the following line should be included in the dump, so that the dump matches the args used for camera calibration

**Comment by SJamieson** [comment]
For closure -- this was fixed when #502  was merged

**Comment by JWhitleyWork** [comment]
Thanks!
